src/client/pydaos/pydaos_core.py

```diff
@@ -107,12 +107,24 @@ def get(self, name):
     def __getitem__(self, name):
         return self.get(name)
 
-    def dict(self, name, v: dict = None):
+    def _get_object_id(self, cid):
+        """ Get the existing DAOS object class ID based on name.  """
+
+        # Default to OC_UNKNOWN (0), which will automatically select an object class.
+        if cid == "0":
+            return 0
+        if cid != "0":
+            return ObjClassID[cid].value
```

Comment:
This will simplify and resolve the pylint error
```suggestion
        if cid == "0":
            return 0
        return ObjClassID[cid].value
```